classes/local/manager/workflow_manager.php

```diff
@@ -249,7 +249,7 @@ public static function activate_workflow($workflowid) {
      */
     private static function reset_manual_workflow_cache() {
```

Comment:
I think it would be a good idea to change the name of this method as well?

Reply:
You're right of course, thanks!